oss-internship-2020/libtiff/README.md

```diff
@@ -0,0 +1,22 @@
+# sapi-libtiff
+Copyright 2020 Google LLC.
+
+## Start use
+You should make sure the libtiff submodule is cloned.
+
+`git clone --recursive https://github.com/alexelex/sapi-libtiff`
+
+## Usage
+
+#### build:
+`mkdir -p build && cd build && cmake .. -DSAPI_ROOT=$HOME/sapi_root -DBUILD_SHARED_LIBS=OFF`
```

Comment:
Instead of `$HOME/sapi_root` I'd go with `/path/to/sapi_root` to make it more clear that this needs to be changed by the person building the project